api/RTCIceCandidate.json

```
@@ -78,23 +78,23 @@
             "firefox": [
               {
                 "version_added": "44",
-                "notes": "Before Firefox 68, the constructor's <code>options</code> parameter was required."
+                "notes": "Before Firefox 68, the constructor's <code>candidateInfo</code> parameter was required."
```

Change the notes like this, I think.

```suggestion
                "notes": "Before Firefox 68, the constructor's <code>candidateInfo</code> parameter is required and the <code>candidateInfo</code> object must have a <code>candidate</code> property."
```